docs/ldml/tr35-keyboards.md

```diff
@@ -308,6 +308,279 @@ For extensibility, the `<special>` element will be allowed at nearly every level
 
 See [Element special](tr35.md#special) in Part 1.
 
+## Normalization
+
+Unicode Normalization, as described in [The Unicode Standard](https://www.unicode.org/reports/tr41/#Unicode/), is a process by which Unicode text is processed to eliminate unwanted distinctions.
```

Comment:
Is this meant to be a link to UNICODE NORMALIZATION FORMS ?
```suggestion
Unicode Normalization, as described in [The Unicode Standard](https://www.unicode.org/reports/tr15/#Unicode/), is a process by which Unicode text is processed to eliminate unwanted distinctions.
```

Reply:
No, it's a link to Unicode itself.